content/reference/current/installation/configuration/includes/frontline.conf

```diff
@@ -24,25 +24,35 @@ frontline {
     #superAdminPassword = DO_NOT_EDIT_FRONTLINE_WILL_REPLACE_THIS
     #secretKey = DO_NOT_EDIT_FRONTLINE_WILL_REPLACE_THIS
   }
+  cassandra-defaults {
+    host = 127.0.0.1
+    host = ${?FRONTLINE_CASSANDRA_HOST}
+    port = 9042
+    port = ${?FRONTLINE_CASSANDRA_PORT}
+    contact-point = ${frontline.cassandra-defaults.host}":"${frontline.cassandra-defaults.port}
+    contact-point = ${?FRONTLINE_CASSANDRA_CONTACT_POINT}
+  }
```

Comment:
????
Should be at top level, shouldn't be?
(here it is inside `frontline`)

Or what do I miss to understand the PR?

Reply:
Oh! Indeed! It was intended to be used for defining environment variable.